portal/views/scheduled_job.py

```diff
@@ -1,15 +1,16 @@
 """Views for Scheduled Jobs"""
 from flask import abort, jsonify, Blueprint, request
-from flask import render_template
+from flask import render_template, current_app
 from flask_user import roles_required
 
 from ..audit import auditable_event
 from ..database import db
-from ..extensions import celery, oauth
+from ..extensions import oauth
 from ..models.role import ROLE
 from ..models.scheduled_job import ScheduledJob
 from ..models.user import current_user
 
+from ..factories.celery import create_celery
```

Comment:
can we keep the imports alphabetized?  there's a section for external followed by a line of whitespace and then the locals.